### ISSUE TITLE
installation issues with latest github actions windows 2022 runner

### ISSUE THREAD

**Issue by flavorjones** [net-ssh/bcrypt_pbkdf-ruby#24]
In the last few days, Github updated their Windows 2022 (`windows-latest`) runner, and it looks like something in there has broken the bcrypt_pbkdf build.

The build error I'm seeing is:

```
current directory:
C:/hostedtoolcache/windows/Ruby/3.2.4/x64/lib/ruby/gems/3.2.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
make.exe DESTDIR\= sitearchdir\=./.gem.20240516-4380-2nneud
sitelibdir\=./.gem.20240516-4380-2nneud
generating bcrypt_pbkdf_ext-x64-mingw-ucrt.def
compiling bcrypt_pbkdf.c
compiling bcrypt_pbkdf_ext.c
compiling blowfish.c
compiling explicit_bzero.c
explicit_bzero.c: In function 'explicit_bzero':
explicit_bzero.c:18:9: error: implicit declaration of function 'bzero'
[-Wimplicit-function-declaration]
   18 |         bzero(p, n);
      |         ^~~~~
explicit_bzero.c:18:9: warning: incompatible implicit declaration of built-in
function 'bzero' [-Wbuiltin-declaration-mismatch]
make: *** [Makefile:248: explicit_bzero.o] Error 1
```

I'm seeing this in the integration tests for rails/tailwindcss-rails.

- Passing build from May 8: https://github.com/rails/tailwindcss-rails/actions/runs/8998333655/job/24718244606
- Failing build from this morning: https://github.com/rails/tailwindcss-rails/actions/runs/9092067481/job/25058810906#step:4:564

The runner images are described here:

- Passing: https://github.com/actions/runner-images/blob/win22/20240421.1/images/windows/Windows2022-Readme.md
- Failing: https://github.com/actions/runner-images/blob/win22/20240514.3/images/windows/Windows2022-Readme.md

I'm sorry I'm not able to narrow this down any further today.

**Comment by mfazekas** [comment]
@flavorjones can you try 1.1.1.rc2? bzero was replace with memset

**Comment by flavorjones** [comment]
@mfazekas Looks good! Green build at https://github.com/rails/tailwindcss-rails/actions/runs/9149629566/job/25153469345?pr=364

Thank you!

**Comment by mfazekas** [comment]
released 1.1.1 final
